src/System.IO.Pipelines/src/System/IO/Pipelines/ThrowHelper.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.IO.Pipelines
+{
+    internal class ThrowHelper
```

Comment:
Declare as "internal static class" to avoid creation of unnecessary constructor.